PlaywrightUltimate/.auth/state.json

```diff
@@ -0,0 +1 @@
main\u0022:\u0022courses.ultimateqa.com\u0022,\u0022$current_url\u0022:\u0022https://courses.ultimateqa.com/enrollments\u0022,\u0022$browser_version\u0022:127,\u0022$screen_height\u0022:1080,\u0022$screen_width\u0022:1920,\u0022mp_lib\u0022:\u0022web\u0022,\u0022$lib_version\u0022:\u00222.55.0\u0022,\u0022$insert_id\u0022:\u0022tswd9xzk697pdxxw\u0022,\u0022time\u0022:1722711505.078,\u0022distinct_id\u0022:\u0022lichar@ukr.net\u0022,\u0022$device_id\u0022:\u0022191199b6ca56ec-06681d06d5ccfa-26001e51-1fa400-191199b6ca56ec\u0022,\u0022$initial_referrer\u0022:\u0022$direct\u0022,\u0022$initial_referring_domain\u0022:\u0022$direct\u0022,\u0022$user_id\u0022:\u0022lichar@ukr.net\u0022,\u0022$anon_distinct_id\u0022:\u0022$device:191199b6ca56ec-06681d06d5ccfa-26001e51-1fa400-191199b6ca56ec\u0022,\u0022token\u0022:\u0022cc99cd1e0fd2ae585a556ab2b2402a01\u0022}}}]"},{"name":"__mpq_cc99cd1e0fd2ae585a556ab2b2402a01_pp","value":"[{\u0022id\u0022:\u0022lof6nwvf4lvxdd9o\u0022,\u0022flushAfter\u0022:1722711515076,\u0022payload\u0022:{\u0022$set_once\u0022:{\u0022$initial_referrer\u0022:\u0022$direct\u0022,\u0022$initial_referring_domain\u0022:\u0022$direct\u0022},\u0022$token\u0022:\u0022cc99cd1e0fd2ae585a556ab2b2402a01\u0022,\u0022$distinct_id\u0022:\u0022lichar@ukr.net\u0022,\u0022$device_id\u0022:\u0022191199b6ca56ec-06681d06d5ccfa-26001e51-1fa400-191199b6ca56ec\u0022,\u0022$user_id\u0022:\u0022lichar@ukr.net\u0022}},{\u0022id\u0022:\u0022zq6z7wxl6aheuyq4\u0022,\u0022flushAfter\u0022:1722711515079,\u0022payload\u0022:{\u0022$set\u0022:{\u0022$os\u0022:\u0022Windows\u0022,\u0022$browser\u0022:\u0022Chrome\u0022,\u0022$browser_version\u0022:127,\u0022$email\u0022:\u0022lichar@ukr.net\u0022,\u0022$created\u0022:\u00222024-08-01 21:07:44 \u002B0000\u0022,\u0022$last_login\u0022:\u00222024-08-03 18:51:04 \u002B0000\u0022,\u0022$first_name\u0022:\u0022Maksym\u0022,\u0022$last_name\u0022:\u0022Vakarchuk\u0022,\u0022sign_in_count\u0022:77,\u0022admin\u0022:false,\u0022referral_code\u0022:\u0022\u0022,\u0022tenant\u0022:\u0022UltimateQA\u0022,\u0022subdomain\u0022:\u0022qa-academy\u0022,\u0022Is Paying\u0022:\u0022false\u0022,\u0022Revenue\u0022:\u00220.0\u0022,\u0022Courses\u0022:[]},\u0022$token\u0022:\u0022cc99cd1e0fd2ae585a556ab2b2402a01\u0022,\u0022$distinct_id\u0022:\u0022lichar@ukr.net\u0022,\u0022$device_id\u0022:\u0022191199b6ca56ec-06681d06d5ccfa-26001e51-1fa400-191199b6ca56ec\u0022,\u0022$user_id\u0022:\u0022lichar@ukr.net\u0022}}]"},{"name":"lastExternalReferrer","value":"empty"},{"name":"lastExternalReferrerTime","value":"1722711502381"},{"name":"ajs_anonymous_id","value":"\u002241308675-460f-4776-87b3-3c82ceb4c3cf\u0022"},{"name":"_grecaptcha","value":"09AE23sa5enc_ed1zR1X13GZFqpSjMiN-m2RDB9uwN0V38B6BjvfKFYJq7JIt4U_HC0abNGlx5Ek_5HqflM7eLg7zr_RaDSthggoLDaw"}]},{"origin":"https://www.google.com","localStorage":[{"name":"rc::a","value":"eXRlMnlqOWowNmVk"},{"name":"rc::f","value":"0aAE23sa7rn2x7myXkzC_2MDZuqP_ES20LHX9_j6fjDop3b4vko0asza8gjUaHn1bzQsbJz30"}]}]}
```

Comment:
this file should not be here. How it happened what with .gitignore changes it i present in last commit?

Reply:
It appeared because of one commit for test and gitignore, will fix that